RELEASE_NOTES.md

```diff
@@ -1,3 +1,112 @@
+## Cookstyle 5.2
+
+### New Chef Cops
+
+#### Chef/InvalidPlatformMetadata
+
+The `InvalidPlatformMetadata` cop detects invalid platform names being used in the supports metadata.rb method. It uses a hardcoded set of common typos so it's not able to detect all invalid platforms, but it does detect the most common mistakes found on the Supermarket.
```

Comment:
```suggestion
The `InvalidPlatformMetadata` cop detects invalid platform names in the supports metadata.rb method. It uses a hardcoded set of common typos so it can't detect all invalid platforms, but it does detect the most common mistakes found on the Supermarket.
```